src/initial-submission/components/SubmissionWizard.tsx

```diff
@@ -36,6 +43,9 @@ const SubmissionWizard: React.FC<RouteComponentProps> = ({
         history.location.pathname.split('/')[3] && history.location.pathname.split('/')[3].toLowerCase();
     const getCurrentStepPathIndex = (): number =>
         stepConfig.findIndex((config): boolean => config.id === getCurrentStepPath());
+    const { data, loading } = useQuery<GetSubmission>(getSubmissionQuery, {
+        variables: { id: history.location.pathname.split('/')[3] && history.location.pathname.split('/')[2] },
```

Comment:
lets see if theres an easier way to get id from react-routers props. This could catch us if we change routes